app/controllers/users.js

```diff
@@ -36,7 +36,40 @@ exports.login = (req, res, next) => {
   });
 };
 
-exports.update = (req, res, next) => {
+exports.invalidateAll = (req, res) => {
+  const user = req.user;
+  const SALT_ROUNDS = 10;
+
+  bcrypt.genSalt(SALT_ROUNDS).then((verificationCode) => {
+    user.verificationCode = verificationCode;
+    user.save();
+    res.status(200);
+    res.end();
+  }).catch(() => {
+    res.status(500);
```

Comment:
shouldn't you just throw an exception here?
That way the response will still be 500, and then you can read the logs. This way no logs are generated